### PR TITLE
Some new fonts + small visual modification

### PR DESCRIPTION
**Pull request by superscratch444tre**
Adds some new fonts.

### CODE REVIEW

**Comment by Secret-chest** [comment]
Please add names (and I already have the typewriter one)

File: main.py [hunk 1]
```diff
@@ -6,22 +6,25 @@
...
+print("│9 Blue                 │ 🇦 🇧 🇨 🇩 🇪 🇫 🇬 🇭 🇮 🇯 🇰 🇱 🇲 🇳 🇴 🇵 🇶 🇷 🇸 🇹 🇺 🇻 🇼 🇽 🇾 🇿                                    │▒▒")
+print("│10 Squared             │ 🄰🄱🄲🄳🄴🄵🄶🄷🄸🄹🄺🄻🄼🄽🄾🄿🅀🅁🅂🅃🅄🅅🅆🅇🅈🅉                                                   │▒▒")
+print("│11 Circled             │ 🅐🅑🅒🅓🅔🅕🅖🅗🅘🅙🅚🅛🅜🅝🅞🅟🅠🅡🅢🅣🅤🅥🅦🅧🅨🅩                                           │▒▒")
+print("│12 Small Caps          │ ᴀʙᴄᴅᴇғɢʜɪᴊᴋʟᴍɴᴏᴘǫʀsᴛᴜᴠᴡxʏᴢ                                                          │▒▒")
+print("│13 Cool Handwriting    │ ᗩᗷᑕᗪEᖴGᕼIᒍKᒪᗰᑎOᑭᑫᖇᔕTᑌᐯᗯ᙭Yᘔ                                                      │▒▒")
 print("│14 Reversed            │ AᗺƆᗡƎꟻວHIᒐꓘ⅃MИOꟼϘЯƧTUV                                                              │▒▒")
 print("│15 Upside down         │ ∀ᗺƆᗡƎℲפHIſꓘ˥WNOԀQɹS┴∩ΛMX⅄Zɐqɔpǝɟƃɥᴉɾʞlɯuodbɹsʇnʌʍxʎz                                │▒▒")
+print("│16 Cool text           │ 卂乃匚ᗪ乇千Ꮆ卄丨ﾌҜㄥ爪几ㄖ卩Ɋ尺丂ㄒㄩᐯ山乂ㄚ乙                                         │▒▒")
+print("│17 Cool text           │ 𝙰𝙱𝙲𝙳𝙴𝙵𝙶𝙷𝙸𝙹𝙺𝙻𝙼𝙽𝙾𝙿𝚀𝚁𝚂𝚃𝚄𝚅𝚆𝚇𝚈𝚉𝚊𝚋𝚌𝚍𝚎𝚏𝚐𝚑𝚒𝚓𝚔𝚕𝚖𝚗𝚘𝚙𝚚𝚛𝚜𝚝𝚞𝚟𝚠𝚡𝚢𝚣                           │▒▒")
+print("│18 Font                │ 𝕒𝕓𝕔𝕕𝕖𝕗𝕘𝕙𝕚𝕛𝕜𝕝𝕞𝕟𝕠𝕡𝕢𝕣𝕤𝕥𝕦𝕧𝕨𝕩𝕪𝕫𝔸𝔹ℂ𝔻𝔼𝔽𝔾ℍ𝕀𝕁𝕂𝕃𝕄ℕ𝕆ℙℚℝ𝕊𝕋𝕌𝕍𝕎𝕏𝕐ℤ𝟙𝟚𝟛𝟜𝟝𝟞𝟟𝟠𝟡𝟘              │▒▒")
```
[the remark this answers is not in the document]
Comment (Secret-chest) [reply main.py:27]: 17. Typewriter 2
18. Outline

File: main.py [hunk 2]
```diff
@@ -6,27 +6,34 @@
...
+print("│11 Circled             │ 🅐🅑🅒🅓🅔🅕🅖🅗🅘🅙🅚🅛🅜🅝🅞🅟🅠🅡🅢🅣🅤🅥🅦🅧🅨🅩                                           │▒▒")
+print("│12 Small Caps          │ ᴀʙᴄᴅᴇғɢʜɪᴊᴋʟᴍɴᴏᴘǫʀsᴛᴜᴠᴡxʏᴢ                                                          │▒▒")
+print("│13 Cool Handwriting    │ ᗩᗷᑕᗪEᖴGᕼIᒍKᒪᗰᑎOᑭᑫᖇᔕTᑌᐯᗯ᙭Yᘔ                                                      │▒▒")
 print("│14 Reversed            │ AᗺƆᗡƎꟻວHIᒐꓘ⅃MИOꟼϘЯƧTUV                                                              │▒▒")
 print("│15 Upside down         │ ∀ᗺƆᗡƎℲפHIſꓘ˥WNOԀQɹS┴∩ΛMX⅄Zɐqɔpǝɟƃɥᴉɾʞlɯuodbɹsʇnʌʍxʎz                                │▒▒")
+print("│16 Cool text           │ 卂乃匚ᗪ乇千Ꮆ卄丨ﾌҜㄥ爪几ㄖ卩Ɋ尺丂ㄒㄩᐯ山乂ㄚ乙                                         │▒▒")
+print("│18 Outline             │ 𝕒𝕓𝕔𝕕𝕖𝕗𝕘𝕙𝕚𝕛𝕜𝕝𝕞𝕟𝕠𝕡𝕢𝕣𝕤𝕥𝕦𝕧𝕨𝕩𝕪𝕫𝔸𝔹ℂ𝔻𝔼𝔽𝔾ℍ𝕀𝕁𝕂𝕃𝕄ℕ𝕆ℙℚℝ𝕊𝕋𝕌𝕍𝕎𝕏𝕐ℤ𝟙𝟚𝟛𝟜𝟝𝟞𝟟𝟠𝟡𝟘              │▒▒")
+print("│19 Font                │ ąცƈɖɛʄɠɧıʝƙƖɱŋơ℘զཞʂɬų۷ῳҳყʑ                                                          │▒▒")
+print("│20 Font                │ αɓ૮∂εƒɠɦเʝҡℓɱɳσρզ૨รƭµѵωאყƶ                                                           │▒▒")
+print("│21 Font                │ αႦƈԃҽϝɠԋιʝƙʅɱɳσρϙɾʂƚυʋɯxყȥ                                                          │▒▒")
```
Comment (superscratch444tre) [inline main.py:29]: I cannot find a name for these...

Comment (Secret-chest) [reply main.py:29]: 19. Curly
20. Handwriting 3
21. Strange

**Comment by superscratch444tre** [comment]
@Secret-chest I think it is ready for merge!

**Pull request merged**